ISPP-G5/NexONG_Frontend
# Fix/262 info

In this PR, I have added info for actors in admin page and added the event info when the volunteer clicks on the event.
To test it:
- Log in as admin and see Voluntarios, Voluntarios > Solicitudes, Socios, Educadores, Familia and Familia > Solicitudes
- Log in as volunteer and click on an event in the calendar
